front/app/modules/commercial/content_builder/admin/components/CraftComponents/TwoColumn/index.tsx

```diff
@@ -21,10 +21,15 @@ const TwoColumn: UserComponent = () => {
       display="flex"
       w="100%"
     >
-      <Box flex="1">
+      <Box
+        m="-4px"
+        mb={isLargeTablet ? '2px' : '-2px'}
+        mr={isLargeTablet ? '-4px' : '0px'}
+        flex="1"
```

Comment:
Bit unsure about how I fixed the selected box issue for two-column. On master, when you select two-column, there is a gap between the component border and the selection box border. If there is a better way you suggest for fixing this, just let me know. :)

Reply:
Not a bit fan of this myself either. Do you think it's time to add a `gap` prop to `Box` in the component library?

Reply:
Sounds good!